package.json

```diff
@@ -8,13 +8,16 @@
   "license": "MIT",
   "dependencies": {
     "@types/node": "^11.9.4",
+    "aws-sdk": "^2.439.0",
     "envkey": "^1.2.6",
     "typescript": "^3.4.1"
   },
   "scripts": {
+    "setup": "docker network create localdynamodb",
     "test": "mocha tests/unit/",
     "build": "tsc && cp package.json ./build/package.json && cd build && docker run --rm -v \"$PWD\":/var/task lambci/lambda:build-nodejs8.10 npm install --production",
-    "local": "npm run build && sam local invoke --no-event"
+    "local": "npm run build && sam local invoke --no-event  --docker-network localdynamodb",
+    "local-dynamodb": "docker run -d -v \"$PWD\":/dynamodb_local_db -p 8000:8000 --network localdynamodb --name dynamodb cnadiminti/dynamodb-local"
```

Comment:
Can you combine setup and local-dynamodb together into one command `local-dynamodb`